app/_hub/kong-inc/openid-connect/index.md

```diff
@@ -267,43 +263,118 @@ are used only in some flows, and others are used more broadly. Generally speakin
 `authorization_cookie_domain`        | The `Domain` parameter of the authorization cookie.
 `authorization_cookie_samesite`      | The `Same-Site` parameter of the authorization cookie.
 `authorization_cookie_httponly`      | The `HTTP Only` flag of the authorization cookie.
 `authorization_cookie_secure`        | The `Secure` flag of the authorization cookie.
+`token_endpoint`                     | The url of token endpoint that can be used to override the standard token endpoint announced in discovery document.
+`token_endpoint_auth_method`         | Used to override any defaults or `token_endpoint_auth_methods_supported` from discovery when authenticating on token endpoint.
+`token_headers_names`                | Extra arguments names that you want to include in token endpoint requests.
+`token_headers_values`               | Values for the extra arguments that you want to include in token endpoint requests.
+`token_headers_client`               | When Kong calls token endpoint, you can specify the headers that Kong will pass to token endpoint from client request headers.
+`token_headers_replay`               | When Kong calls token endpoint, you can specify the headers that you want the Kong to send back to the client.
+`token_headers_prefix`               | If you want you can prefix the token endpoint headers with a string to differentiate them from other headers.
+`token_headers_grants`               | You can limit the grants for which the token headers are replayed.
+`token_post_args_names`              | Extra arguments names that you want to include in token endpoint post args.
+`token_post_args_values`             | Values for the extra arguments that you want to include in token endpoint post args.
+`token_post_args_client`             | These parameters are passed from client request to token endpoint.
+`introspection_endpoint`             | The url of introspection endpoint that can be used if the OP doesn't announce a non-standard introspection endpoint in discovery document.
+`introspection_hint`                 | Use this parameter if you want to change introspection request `token_type_hint` argument to something else than the default `access_token`.
+`introspection_headers_names`        | Extra arguments names that you want to include in introspection requests.
+`introspection_headers_values`       | Values for the extra arguments that you want to include in introspection requests.
+`introspection_post_args_names`      | Extra arguments names that you want to include in introspection endpoint post args.
+`introspection_post_args_values`     | Values for the extra arguments that you want to include in introspection endpoint post args.
+`introspection_post_args_client`     | These parameters are passed from client request to introspection endpoint.
+`introspect_jwt_tokens`              | Enable this option to also introspect JWT tokens (and not only those opaque ones). Can be used to check revocation of JWT tokens.
```

Comment:
```suggestion
`introspect_jwt_tokens`              | Enable this option to also introspect JWT tokens (and not only those opaque ones). It can be used to check the revocation of JWT tokens.
```